README.rst

```diff
@@ -84,7 +84,11 @@ This section outlines what stages a proposal may go through. The stage is identi
 
    Acceptance of the proposal implies that the implementation will be accepted
    into GHC provided it is well-engineered, well-documented, and does not
-   complicate the code-base too much.
+   complicate the code-base too much.  Moreover the GHC maintainers may decline
+   to accept (or revert) an implementation pending revisions to the proposal, if
```

Comment:
This is a bit of a tortured sentence, hard to parse even for a native English speaker. Consider rewording.

Reply:
Indeed, I've reworded.